packages/components/core/option/action-container.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'kbq-action-container',
+    exportAs: 'kbqActionContainer',
+    standalone: true,
+    template: `
+        <div class="kbq-action-container__gradient"></div>
+        <div class="kbq-action-container__box">
+            <ng-content />
+        </div>
+    `,
+    styleUrls: ['./action-container.scss'],
+    host: {
+        class: 'kbq-action-container'
+    },
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class KbqActionContainerComponent {}
```

Comment:
```suggestion
export class KbqActionContainer {}
```

в стайл гайде ангуляра пропало упоминание о суффиксах в виде `Component, Directive, Pipe` и тд
мб и мы поступим также?